trunk/SUAVE/Methods/Missions/Segments/Hover/Climb.py

```diff
@@ -47,7 +49,7 @@ def initialize_conditions(segment):
     if alt0 is None:
         if not segment.state.initials: raise AttributeError('initial altitude not set')
         alt0 = -1.0 * segment.state.initials.conditions.frames.inertial.position_vector[-1,2]
-
+   
```

Comment:
remove the spaces/tab

Reply:
I'm going to correct this and delete this pull request per our discussion this morning